### ISSUE TITLE
Missing user 'app'

### ISSUE THREAD

**Issue by stevenvandervalk** [vecnet/deploy-dl#3]
Against master on a fresh vagrant I see the error below.  I'm guessing you don't have this error as you are reusing the same vm with the user already created?  Or perhaps I'm doing something wrong with my vbox, vagrant, ansible setup. 

```
TASK: [noids | create pool storage palce] ************************************* 
failed: [192.168.33.13] => (item=/opt/noids) => {"failed": true, "gid": 0, "group": "root", "item": "/opt/noids", "mode": "0755", "owner": "root", "path": "/opt/noids", "size": 4096, "state": "directory", "uid": 0}
msg: chown failed: failed to look up user app
failed: [192.168.33.13] => (item=/opt/noids/pools) => {"failed": true, "gid": 0, "group": "root", "item": "/opt/noids/pools", "mode": "0755", "owner": "root", "path": "/opt/noids/pools", "size": 4096, "state": "directory", "uid": 0}
msg: chown failed: failed to look up user app

FATAL: all hosts have already failed -- aborting
```

**Comment by dbrower** [comment]
But it installed fine on my vagrant and your other VM.

User is made here: https://github.com/vecnet/deploy-dl/blob/master/roles/vecnet/tasks/main.yml#L2
and here: https://github.com/vecnet/deploy-dl/blob/master/roles/discovery/tasks/main.yml#L3

It seems the user is being made AFTER noids is installed, so that fails. The other times must have worked because the user was created first, and then the noids step was added to the playbook.

I'll rearrange things.

**Comment by dbrower** [comment]
Fixed in 71c5922bfe59fa927a019a667a1cd40132d9b217
